maliput_ros_translation/include/maliput_ros_translation/convert.h

```diff
@@ -192,4 +200,69 @@ maliput::api::LanePosition FromRosMessage(const maliput_ros_interfaces::msg::Lan
 maliput::api::RoadPosition FromRosMessage(const maliput::api::RoadGeometry* road_geometry,
                                           const maliput_ros_interfaces::msg::RoadPosition& road_position);
 
+/// Converts a maliput::api::Rotation into maliput_ros_interfaces::msg::Rotation.
+/// @param rotation The rotation to convert.
+/// @return A maliput_ros_interfaces::msg::Rotation.
+maliput_ros_interfaces::msg::Rotation ToRosMessage(const maliput::api::Rotation& rotation);
```

Comment:
Maybe we don't have any query where the rotation is an input but for completeness isn't it better to provide also a FromRosMessage for the rotations?

Reply:
I preferred to grow this as required. Will add a ticket to complete the translations though.